examples/starwars/lib/reviews/force_rebroadcast.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:graphql_flutter/graphql_flutter.dart';
+
+class ForceRebroadcast extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Mutation(
+      options: MutationOptions(
+        document: r'''
+        mutation Rebroadcast {
+        }
```

Comment:
Is this valid query language?

Reply:
no - I'm removing it as it was just a utility for testing some pagination issues.